components/docker-auth/pkg/extension/acl.go

```diff
@@ -20,239 +20,229 @@ package extension
 			image)
 	}
 	if err != nil {
-		log.Printf("[%s] Error in retrieving the visibility for %s/%s from the database :%s\n", execId,
-			organization, image, err)
-		return visibility, err
+		return visibility, fmt.Errorf("[%s] Error in retrieving the visibility for %s/%s from the "+
+			"database :%s", execId, organization, image, err)
 	}
 	return visibility, nil
 }
 
-func isUserAvailable(db *sql.DB, organization string, user string, execId string) (bool, error) {
-	log.Printf("[%s] Checking whether the user %s exists in the organization %s\n", execId, user, organization)
+func isUserAvailable(db *sql.DB, organization, user string, logger *zap.SugaredLogger, execId string) (bool, error) {
+	logger.Debugf("[%s] Checking whether the user %s exists in the organization %s", execId, user,
+		organization)
 	results, err := db.Query(getUserAvailabilityQuery, user, organization)
 	defer func() {
-		closeResultSet(results, "isUserAvailable", execId)
+		closeResultSet(results, "isUserAvailable", logger, execId)
 	}()
 	if err != nil {
-		log.Printf("[%s] Error while calling the mysql query getUserAvailabilityQuery :%s\n", execId, err)
-		return false, err
+		return false, fmt.Errorf("[%s] Error while calling the mysql query "+
+			"getUserAvailabilityQuery :%s\n", execId, err)
 	}
 	if results.Next() {
-		log.Printf("[%s] User %s is available in the organization :%s\n", execId, user, organization)
+		logger.Debugf("[%s] User %s is available in the organization :%s", execId, user, organization)
 		return true, nil
 	} else {
-		log.Printf("[%s] User %s is not available in the organization :%s\n", execId, user, organization)
+		logger.Debugf("[%s] User %s is not available in the organization :%s", execId, user, organization)
 		return false, nil
 	}
 }
 
-func isAuthorizedToPull(db *sql.DB, user string, organization string, image string, execId string) (bool, error) {
-	log.Printf("[%s] ACL is checking whether the user %s is authorized to pull the image %s in the "+
-		" organization %s.\n", execId, user, image, organization)
+func isAuthorizedToPull(db *sql.DB, user string, organization string, image string,
+	logger *zap.SugaredLogger, execId string) (bool, error) {
+
+	logger.Debugf("[%s] ACL is checking whether the user %s is authorized to pull the image %s in the "+
+		" organization %s.", execId, user, image, organization)
 
-	visibility, err := getImageVisibility(db, image, organization, execId)
+	visibility, err := getImageVisibility(db, image, organization, logger, execId)
 
 	if err != nil {
-		log.Printf("[%s] User %s is not authorized to pull the image %s/%s.\n", execId, user,
+		logger.Debugf("[%s] User %s is not authorized to pull the image %s/%s.", execId, user,
 			organization, image)
-		return false, fmt.Errorf("error occured while geting visibility of image %s/%s", organization, image)
+		return false, fmt.Errorf("error occured while geting visibility of image. User %s is not "+
```

Comment:
Shall we use one convention ? Either start with capital or sample

Reply:
Fixed by #159